### PR TITLE
Fix python3 issues with deploy_docker.

### PR DESCRIPTION
**Pull request by manthey**
This has been tested with start, provision, stop, and remove.

This fixes the issue reported in #379.

### CODE REVIEW

File: ansible/utils/cli_test.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def test_cli(client, folder, opts):
         if item['name'].startswith('TCGA-02'):
             testItem = item
             break
-    localFile = client.listFile(testItem['_id']).next()
+    localFile = next(client.listFile(testItem['_id']))
```
Comment (zachmullen) [inline ansible/utils/cli_test.py]: The py2/3 safe way to call this would be `six.next(...)`

Comment (manthey) [reply ansible/utils/cli_test.py]: I thought `next` worked the same in Python 2 and Python 3 provided Python >= 2.6.

Comment (zachmullen) [reply ansible/utils/cli_test.py]: I have encountered situations in the past where `next` did not work but replacing with `six.next` did, but those may have been weird edge cases of iterators or something. If this works, we can go with it.

**Comment by manthey** [comment]
We are assuming that the config file is written in utf8, which is the same assumption made in Girder.  For Girder, in general, I think we *want* to assume this.  Otherwise, we need to expose some method to know what the config file is encoded in (which would go in a config file...).

**Pull request merged**